spec/factories/responses.rb

```diff
@@ -3,10 +3,11 @@
 # Table name: responses
 #
 #  id             :bigint           not null, primary key
+#  not_working    :boolean          default(FALSE)
```

Comment:
Please create a variant factory that recreates this scenario (and test it in response_spec.rb). Using [traits](https://thoughtbot.com/blog/never-fear-traits-are-here) is a good approach, so it could be generated by `let(:response) { create(:response, :did_not_work) }`.